string[] texts # list of input labels for object detector
---
DetectedObject[] objects